hail/python/hail/expr/types.py

```diff
@@ -653,9 +653,16 @@ def _convert_from_json(self, x):
 
     def _convert_to_json(self, x):
         data = x.reshape(x.size).tolist()
+
+        strides = []
+        running_shape_product = x.itemsize
+        for shape_element in reversed(x.shape):
```

Comment:
`shape_element` is `dimension_size` or `axis_size` right? Either feels a bit more evocative than `shape_element`.